README.md

```diff
@@ -1243,23 +1247,29 @@ in /var/lib/pcsd with the file name FILENAME.crt and FILENAME.key, respectively.
                 value: '5'
               - name: interval
                 value: '1min'
-      - id: dummy-1
+      - id: dmy-1
```

Comment:
`dmy` is still close to `dummy` (although it does pass `woke`).  I would prefer a better name like `resource-1` or `example-1` or something like that.